src/Components/SmartComponents/CVEs/CVEsTable.js

```diff
@@ -49,7 +49,7 @@ const CVEsTableWithContext = ({ context, header, entity }) => {
     const { cves, methods, selectedCves, openedCves } = context;
     const isEmpty = cves.data.length === 0;
 
-    const rows = cves.data
+    const rows = cves.data && cves.data
```

Comment:
great catch! 

you can use  new syntax `?.`

Reply:
I tried, but babel plugin, from some reason compiled the .? operator as $ and it did not work, I was testing this on top of #625  which changes the babel plugin config and that might be it.